### PR TITLE
Make counter records have new capture pvi group

### LINKED ISSUE #57: New PVI group for counters
Move the fields from here

image

into a new group on counters, and also add a button that displays the counter in the Positions table

### PR DESCRIPTION
**Pull request by evalott100**
Closes #57

### CODE REVIEW

**Comment by evalott100** [comment]
The `CAPTURE, OFFSET, SCALE, UNITS` are in `PARAMETERS` if the field is not a `COUNTER`:
image

and in `OUTPUTS if the field is a `COUNTER`:
image

**Comment by evalott100** [comment]
The `pos_out` pvi screens now have a button to open the table, and `CAPTURE, OFFSET, SCALE, UNITS` have been put in a new `CAPTURE` group.
image

The table has been given a name field
image

This requires the following changes to pvi (and merge of the device-ref branch):

* A way to change this to `True`
https://github.com/epics-containers/pvi/blob/6a51e6728857a8cd36415afbe03bd6e71aeee032/src/pvi/_format/screen.py#L431

* A way to ignore this assert so the "Name" column can be added
https://github.com/epics-containers/pvi/blob/6a51e6728857a8cd36415afbe03bd6e71aeee032/src/pvi/_format/screen.py#L416-L418

* A way for the columns to be aligned given the non-pv device name cell

**Comment by evalott100** [comment]
TOP.bob now correctly displays buttons (which work for navigation).

Screenshot_2023-10-20_15-54-15

**Comment by evalott100** [comment]
Changed the table back to what it was before for this branch

image

**Comment by evalott100** [comment]
New group and index.bob:
image
image

**Comment by evalott100** [comment]
No longer splitting SingnalRW

image

File: src/pandablocks_ioc/_pvi.py [hunk 1]
```diff
@@ -256,4 +255,19 @@ def create_pvi_records(record_prefix: str):
...
+                    device_ref = DeviceRef(
+                        "AllPostionCaptureParameters",
+                        "CAPTURE",
+                        "PandA_POSITIONS_TABLE",
+                    )
+                    for group in device.children:
+                        if group.name == "CAPTURE":
+                            cast(Group, group).children.append(device_ref)
+
                 formatter.format(device, record_prefix + ":", bobfile_path)
```
Comment (coretl) [inline src/pandablocks_ioc/_pvi.py:273]: I'm not clear what this code is doing...

Comment (evalott100) [reply src/pandablocks_ioc/_pvi.py]: I'm hoping it's a lot clearer now!

**Review by coretl — APPROVED** [review]

**Pull request merged**